pkg/test/context.go

```diff
@@ -64,6 +65,7 @@ func (f *Framework) newTestCtx(t *testing.T) *TestCtx {
 		client:            f.Client,
 		kubeclient:        f.KubeClient,
 		restMapper:        f.restMapper,
+		cleanupOnError:    f.cleanupOnError,
```

Comment:
```suggestion
		skipCleanupError:    f.skipCleanupError,
```

WDYT about a name more intuitive?

Reply:
sounds good to me!